bundle-workflow/src/read_write_s3.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import boto3
+
+
+class ReadWriteFiles:
+
+    # get_S3Objects:Used for getting the files from S3
+    # bucket_name: Name of the bucket you want to get the file from
+    # role_arn: Amazon Resource Name (ARN) of the role that you want to assume
+    # role_session_name: Name for the session
+    # bucket_bundle: Object path from where you want to get the bundle
+    # bucket_manifest: Object path from where you want to get the manifest
+    # path_to_save_bundle: Path where you want to save the bundle at
+    # path_to_save_manifest: Path where you want to save the manifest at
```

Comment:
Could you add these comments as pydoc 

"""
"""

Reply:
I have added them in the 
"""
"""
pydoc block.